### ISSUE TITLE
Tests execution pauses when browser window is not visible

### ISSUE THREAD

**Issue by AlexKamaev** [DevExpress/testcafe-browser-tools#158]
based on https://github.com/DevExpress/testcafe/issues/2939

> The issue is related to Сhrome background tab behavior optimization. Please refer to the following article: https://developers.google.com/web/updates/2017/03/background_tabs. You can use the --disable-background-timer-throttling command line argument to prevent this behavior. So, your CLI command will look like testcafe "chrome:--disable-background-timer-throttling" test.js`

**Comment by tbcoyle** [comment]
As an FYI, it looks like chrome occlusion tracking now works on windows:
https://blog.chromium.org/2020/11/tab-throttling-and-more-performance.html

As a result, I am now encountering the issue described in DevExpress/testcafe#2939 on Windows 10 / Chrome Version 87.0.4280.66.

As suggested in the referenced issue, I attempted to fix the problem by using the `--disable-background-timer-throttling` flag, but, despite updating the flag, testcafe execution still paused when the browser window was not visible. Due to the new occlusion tracking, I also had to add the `--disable-backgrounding-occluded-windows` chrome flag, which fixed the issue. For anyone encountering a similar problem on windows, you might try adding that second chrome flag to see if that fixes your issue.
